lib/mauth/faraday.rb

```diff
@@ -36,9 +36,16 @@ def initialize(request_env)
       end
 
       def attributes_for_signing
-        request_url = @request_env[:url].path
-        request_url = '/' if request_url.empty?
-        @attributes_for_signing ||= { verb: @request_env[:method].to_s.upcase, request_url: request_url, body: @request_env[:body] }
+        @attributes_for_signing ||= begin
+          request_url = @request_env[:url].path
+          request_url = '/' if request_url.empty?
```

Comment:
Kind of weird it is called `request_url` when it is only the path
Also what about 
`request_path = @request_env[:url].path.empty? '/' : @request_env[:url].path` instead of two lines

Reply:
Agreed that `request_path` would be better. My concern though was that some people may be manually signing requests as detailed in the README [here](https://github.com/mdsol/mauth-client-ruby#other-request-and-response-signing). Then updating mauth-client would break their usage until they change from `request_url` to `request_path`.

Reply:
I think I understand now.
 There is this method called `attributes_for_signing` here but in fact is more like `potential_attributes_we_may_use_for_signing`. Each version will somehow filter the ones not needed